src/Data/Wrap.agda

```diff
@@ -0,0 +1,31 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Turn a relation into a record definition so as to remember the things
+-- being related.
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Data.Wrap where
+
+open import Data.Nat
+open import Data.Product.Nary.NonDependent
```

Comment:
Strictly, I don't need it at all, because `n`'s type always gets synthesised. What's the best thing to do: still import `Data.Nat.Base` or drop it completely?

Reply:
I would just drop it.